### ISSUE TITLE
Allow Reordering Apps

### ISSUE THREAD

**Issue by Surenix** [rpetrich/ProSwitcher#16]
Please add a toggle to hide the default backgrounded apps (phone, mail, Safari). I already have all 3 on my dock and I personally would prefer to not have to swipe past those to get to the apps I'm really interested in.

Another idea would be a way to re-order apps in the list. Might even be enough to just swap the order so the most recently opened app is first. Then you don't have to swipe through phone, mail, and Safari every time to get to the apps you really care about.

(edit: changed title from Hide default apps and/or re-order them)

**Comment by rpetrich** [comment]
ProSwitcher 0.7 will have the ability to "Show Default Apps" (or to not show them)

Reordering apps is a feature we'd like to implement, but I don't see that happening in the near future. Can you post this as a separate feature request (if it hasn't been already?)

**Comment by grp** [comment]
Yeah; re-ordering like the pre would be awesome but HARD to implement.
